addons/purchase/models/purchase_order.py

```diff
@@ -1076,3 +1077,32 @@ def _is_readonly(self):
         """
         self.ensure_one()
         return self.state == 'cancel'
+
+    def action_create_bill(self):
+        if self.env.user.has_group('purchase.group_down_payment'):
+            return {
+                'name': _('Create bills'),
+                'type': 'ir.actions.act_window',
+                'view_mode': 'form',
+                'res_model': 'purchase.advance.payment.wizard',
+                'target': 'new',
+            }
+        return self.action_create_invoice()
+
+    def _filter_subrecords(self):
+        return lambda x: self == x.purchase_order_id
+
+    def _subrecord_type(self):
+        return self.env['purchase.order.line']
+
+    def _is_locked(self):
+        return self.state == 'done'
+
+    def _create_invoices(self, grouped=False, final=False, date=None):
+        return self.action_create_invoice(True)
+
+    def _qty_field_name(self):
+        return 'product_qty'
```

Comment:
product_qty and product_uom_qty should be merged and declared on the parent class.